RealmSwift/List.swift

```diff
@@ -626,6 +626,10 @@ public final class List<T: Object>: ListBase {
         super.init(array: RLMArray(objectClassName: (T.self as Object.Type).className()))
     }
 
+    internal init(rlmArray: RLMArray) {
+        super.init(array: rlmArray)
```

Comment:
Does this change need to be made in both Swift 2.x and 3.0 portions of the file?